app/models/avatar.rb

```diff
@@ -0,0 +1,23 @@
+class Avatar
+  def initialize(user)
+    @user = user
+  end
+
+  def gravatar(size: 512, style: :retro)
+    "https://www.gravatar.com/avatar/#{ email_digest }?s=#{ size }&d=#{ style }"
+  end
+
+  def url(size: 512)
```

Comment:
Unused method argument - `size`. You can also write as `url(*)` if you want the method to accept any arguments but don't care about them.